github/provider.go

```diff
@@ -157,6 +164,7 @@ func init() {
 		"app_auth.id":              "The GitHub App ID.",
 		"app_auth.installation_id": "The GitHub App installation instance ID.",
 		"app_auth.pem_file":        "The GitHub App PEM file contents.",
+		"write_delay_ms":           "Amount of time in milliseconds to sleep in between writes to GitHub REST API.",
```

Comment:
```suggestion
		"write_delay_ms":           "Amount of time in milliseconds to sleep in between writes to GitHub API.",
```

Does it apply to both REST and GraphQL API types? I believe this gets invoked whenever we detect a mutating HTTP verb.

Also may want to mention our default.

Reply:
Thanks, I've incorporated your suggestion and also added the default (same as before, 1 second in between requests).